src/app/application.cpp

```diff
@@ -71,7 +72,23 @@
 #include "base/bittorrent/session.h"
 #include "base/bittorrent/torrenthandle.h"
 
-static const char PARAMS_SEPARATOR[] = "|";
+namespace
+{
+#define FILELOGGER_SETTINGS_KEY(name) "FileLogger/" name
+    const QString KEY_FILELOGGER_ENABLED = FILELOGGER_SETTINGS_KEY("Enabled");
```

Comment:
@sledgehammer999 , I don't think that without namespace the code will be better. I, personally, like anonymous namespace because they make intents exceptionally clear. One will not think whether there are any `extern` declarations, declarations in a header file, or so.